### DISCUSSION TITLE
The old `(and N more errors)` problem is back.

### DISCUSSION THREAD

**Discussion by andrey-helldar** [Laravel-Lang/common#174] [General]
@Laravel-Lang/laravel-lang, hey!

In Laravel 11, the translation of the phrases `(and N more error)`, `(and N more errors)` and `The given data was invalid.` has been disabled.

It was not possible to write tests using the quick method, because there is no binding with Translator despite the requirement to install it in `composer.json`.

So I opened the [issue](https://github.com/laravel/framework/issues/50534) and will wait for the test result.

**Comment by andrey-helldar** [comment]
They'll fix it soon :)

https://github.com/laravel/framework/pull/50546

**Reply by andrey-helldar** [reply]
Or not...

https://github.com/laravel/framework/pull/50556

**Reply by andrey-helldar** [reply]
Or yes... 😂

https://github.com/laravel/framework/pull/50546

**Reply by Oleksandr-Moik** [reply]
:rofl:

**Reply by andrey-helldar** [reply]
@Laravel-Lang/laravel-lang, It's like some kind of holiday!)

https://github.com/laravel/framework/pull/50560

**Reply by Oleksandr-Moik** [reply]
Напишу як є - це супер! Дійсно як якесь св'ято!) Вітаю! :confetti_ball:

**Reply by andrey-helldar** [reply]
Сам не очікував. Без будь-яких запитань взяти і мовчки прийняти PR. Тейлор в ударі) Я приємно здивований :)
